webrtc/RTCPeerConnection-createAnswer.html

```diff
@@ -0,0 +1,106 @@
+<!doctype html>
+<meta charset=utf-8>
+<title>RTCPeerConnection.prototype.createAnswer</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="RTCPeerConnection-helper.js"></script>
+<script>
+  'use strict';
+
+  // Test is based on the following editor draft:
+  // https://w3c.github.io/webrtc-pc/archives/20170515/webrtc.html
+
+  /*
+   * 4.3.2. createAnswer()
+   */
+
+  /*
+   *  4.1.  If connection's remoteDescription is null return a promise rejected with a
+   *        newly created InvalidStateError.
+   */
+  promise_test(t => {
+    const pc = new RTCPeerConnection();
+    return promise_rejects(t, 'InvalidStateError',
+      pc.createAnswer());
+  }, 'createAnswer() with null remoteDescription should reject with InvalidStateError');
+
+  promise_test(t => {
+    const pc = new RTCPeerConnection();
+
+    return generateOffer({ video: true })
+    .then(offer =>
+      pc.setRemoteDescription(offer)
+      .then(() => pc.createAnswer(offer)))
+    .then(answer => {
```

Comment:
Oh yeah, I got mixed up and createAnswer indeed does not require the offer as argument. Thanks for spotting!